packages/terra-card/src/CardBody.module.scss

```diff
@@ -13,4 +13,5 @@
   .center-content {
     text-align: center;
   }
+  /* stylelint-enable */
```

Comment:
We don't need to disable `selector-class-pattern`, let's remove these stylelint overrides